src/pagination/pagination.service.js

```diff
@@ -0,0 +1,24 @@
+(function () {
```

Comment:
Hmm..yeah, as the comment in there says it's a placeholder because we may need a service to hold variables to hold the current pagination configs (pageSize, pageNum, etc..) when switching between List, Card, and Table views, but I won't know until I integrate them.

I'll remove the service for now and add it later if needed -thanks